packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -180,32 +181,110 @@ async function defaultCreateFeedItems({
   );
 }
 
+/**
+ * @description addXmlStyleSheet appends a xsl stylesheet to the generated xml feed
+ * @param feedDetails the feed content
+ * @param generatePath path where the file would be copied in website
+ * @param xsltFile path to the xslt file
+ * @param contentPaths path to the content directory
+ */
+async function transformFeedWithStylesheet({
```

Comment:
This function doesn't do what it claims to do.

This doesn't only transform the feed, but also perform IO operations (reading xslt/css, writing them to the build folder)